certified-connectors/DocuSign/apiProperties.json

```diff
@@ -5,7 +5,7 @@
           "type": "oauthSetting",
           "oAuthSettings": {
             "identityProvider": "DocuSign",
-            "clientId": "bc0cb699-7d9f-4d83-80bc-f31c5ca820c7",
+            "clientId": "<<Please add your clientID here>>",
```

Comment:
Is this intentional?

Reply:
Yes. There is no need to expose the client id in the config as it is configured using the ISV studio tool.